docs/guides/tutorials/trpc.rst

```diff
@@ -0,0 +1,468 @@
+.. _ref_guide_trpc:
+
+Integrating EdgeDB with tRPC
+============================
+
+This guide explains how to integrate **EdgeDB** with **tRPC** for a modern, 
+type-safe API. We'll cover setting up database interactions, API routing, 
+and implementing authentication, all while ensuring type safety across the 
+client and server.
+
+You can reference the following repositories for more context:
+
+- `TurboRepo Template <https://github.com/scotttrinh/turbo-edb-app/blob/main-template/>`_: A monorepo template using **TurboRepo**, **EdgeDB**, and **tRPC**.
```

Comment:
Maybe we should move or rebuild this into our official examples repo?

Reply:
Or maybe change it to an official template so that people can click "use it as a template"?

Reply:
Yeah, maybe both? Can a subdirectory be a template? Probably not. I certainly don't mind linking to a personal repo of mine, but it feels like it's worth trying to make it a bit more official if we can. I guess since we already have the next+edb-auth template, there is already precedent for having a template repo in the edgedb org. I'll see about transferring it as-is and making it a template.